### PR TITLE
Add email lists to MeSerializers

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

File: lego/apps/email/serializers.py [hunk 1]
```diff
@@ -12,6 +12,23 @@
...
+    class Meta:
+        model = EmailList
+        fields = (
+            "id",
+            "users",
+            "name",
+            "email",
+            "groups",
+            "group_roles",
+            "require_internal_address",
```
Comment (LudvigHz) [inline lego/apps/email/serializers.py:28]: Do you need all these fields? I have a feeling the `/me` response is getting quite large now :upside_down_face:.

**Review by LudvigHz — APPROVED** [review]
Nice :100:

File: lego/apps/email/serializers.py (hunk 1 above)
Comment (odinuge) [reply lego/apps/email/serializers.py:28]: Nah, not that important i guess. But they are quite small however, so waaay smaller than memberships and groups...

Comment (odinuge) [reply lego/apps/email/serializers.py:28]: Also, this drops `/users/me` from 20 to 15 queries in dev, so it is probably faster now.

Comment (odinuge) [reply lego/apps/email/serializers.py:28]: Can probably move stuff out of `/me` to separate endpoints, but perms are pretty nice when using it, and we still "need" all the info when looking at the user profile on the webapp.

Comment (odinuge) [reply lego/apps/email/serializers.py:28]: Also, this only return ids, and not complete user and group entities, so fairly small.

**Review by odinuge — COMMENTED** [review]
Lets test in staging

File: lego/apps/email/serializers.py (hunk 1 above)
Comment (LudvigHz) [reply lego/apps/email/serializers.py:28]: Hmm, yeah we prob have larger endpoints, so I guess it's fine :+1:

**Pull request merged**